### PR TITLE
add mailcatcher for docker-compose envs

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Add mailcatcher in docker-compose envs for easy debugging of mail notifications

### CODE REVIEW

**Review by cutwater — APPROVED** [review]

File: tools/docker/docker-compose.stage.yml [hunk 1]
```diff
@@ -148,6 +148,12 @@ services:
   redis:
     image: "docker.io/redis:alpine"
 
+  mailcatcher:
```
Comment (mkanoor) [inline tools/docker/docker-compose.stage.yml:151]: @Alex-Izquierdo do we need an equivalent one for minikube?

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose.stage.yml:151]: I wasn't sure, but if you consider it, I can add it too.

Comment (mkanoor) [reply tools/docker/docker-compose.stage.yml:151]: @Alex-Izquierdo Should we be modifying these https://github.com/ansible/pinakes/blob/6c77e45523ec662f2e13ee87b08a59d9e7d61be4/tools/keycloak_setup/qa.yml#L47 to point to this mailserver

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose.stage.yml:151]: @mkanoor not at all. Just use that smtp server and catches all mails.

**Review by mkanoor — APPROVED** [review]

**Pull request merged**